draft-ietf-tls-rfc8446bis.md

```diff
@@ -3855,6 +3855,9 @@ safely encrypted under a given set of keys.  {{AEAD-LIMITS}} provides
 an analysis of these limits under the assumption that the underlying
 primitive (AES or ChaCha20) has no weaknesses. Implementations SHOULD
 do a key update as described in {{key-update}} prior to reaching these limits.
+Note that it is not possible to perform a KeyUpdate for early data
+and therefore implementations SHOULD not exceed the limits
+when sending early data.
```

Comment:
I might prefer:

> therefore implementations MUST stop sending early data before exceeding these limits.

No point in being all "SHOULD"y when it comes to this.

FWIW, I worked out that you can exceed the key usage limits with AES and early data.  You need an awful lot of it, but it is at least theoretically possible with one record per byte.  2^32 bytes > 2^24.5 records.